SC 7000071044/NCBI SC S288C/SC 7000071044 NCBI SC S288C output.fasta

```diff
@@ -0,0 +1,28 @@
+>NC_001224|Q0020|6545-8194|forward|SGD:S000007287|15S ribosomal RNA| 
```

Comment:
I can't see this identifier ("Q0020") in the genbank file - how does "SDRNA_RNA_Mining" find this rRNA?

Reply:
I looked for rRNA in the Genbank feature. This is from another gb file ([chrmt.gb](http://sgd-archive.yeastgenome.org/sequence/S288C_reference/NCBI_genome_source/)) that was not successfully pushed. I will fix that. But it is under Key: locus_tag, Value: ['Q0020'] as one of the qualifiers. It might be easier to see the jupyter notebook onscreen outputs:

> type: rRNA
> location: [6545:8194](+)
> qualifiers:
>     Key: GO_component, Value: ['GO:0005763 - mitochondrial small ribosomal subunit [Evidence IDA] [PMID 6262728]']
>     Key: GO_function, Value: ['GO:0003735 - structural constituent of ribosome [Evidence IC] [PMID 6262728]']
>     Key: GO_process, Value: ['GO:0032543 - mitochondrial translation [Evidence IC] [PMID 6262728]']
>     Key: db_xref, Value: ['SGD:S000007287']
>     Key: gene, Value: ['15S_RRNA']
>     Key: locus_tag, Value: ['Q0020']
>     Key: note, Value: ['Ribosomal RNA of the small mitochondrial ribosomal subunit; MSU1 allele suppresses ochre stop mutations in mitochondrial protein-coding genes']
>     Key: product, Value: ['15S ribosomal RNA']

Reply:
Excellent - that explains why I couldn't find it. Great!